profile/gperf.go

```diff
@@ -0,0 +1,132 @@
+package profile
```

Comment:
`PerfStat.Stop` may accidentally interrupt the program if I use the name `perf.go` in my previous implementation since it kills the process by its name. Now, we don't need to kill `perf` and I can change it back. g is for go.